src/Entity/Transformer.php

```diff
@@ -150,7 +150,9 @@ public function load(SnapshotInterface $snapshot)
         $page->setJavascript($content['javascript']);
         $page->setStylesheet($content['stylesheet']);
         $page->setRawHeaders($content['raw_headers']);
-        $page->setTitle($content['title']);
+        if (isset($content['title'])) {
+            $page->setTitle($content['title']);
```

Comment:
On 4.x we are doing different: https://github.com/sonata-project/SonataPageBundle/blob/4.x/src/Entity/Transformer.php#L142